scripts/docker/jenkinsAgent/Jenkinsfile

```diff
@@ -0,0 +1,72 @@
+#!groovy
+
+import org.folio.Constants
+import org.folio.models.FolioModule
```

Comment:
Following packages are not used within this code:

import org.folio.models.FolioModule
import org.folio.models.InstallRequestParams
import org.folio.models.OkapiTenant
import org.folio.models.RancherNamespace
import org.folio.rest_v2.Main
import org.folio.utilities.GitHubClient

Reply:
removed extra libs